nodewatcher/plugins/slurm.py

```diff
@@ -14,52 +14,50 @@
 import shlex
 import os
 
+from common.slurm import PENDING_RESOURCES_REASONS
+
 log = logging.getLogger(__name__)
 
 
+def _run_command(command):
+    try:
+        if isinstance(command, str):
+            command = shlex.split(command)
+        return subprocess.check_output(command, env=dict(os.environ), universal_newlines=True)
```

Comment:
`check_output` is not available in python 2.6.

You can use `from future.moves.subprocess import check_output`
http://python-future.org/compatible_idioms.html#subprocess-check-output

Reply:
👍👍👍 Thanks!!!